tests/cache/models/samples/upsertBulk.js

```diff
@@ -342,7 +343,7 @@ describe('api::redisEnabled::POST::bulkUpsert ' + path, () => {
           expect(res.body[1].value).to.equal('10');
           done();
         });
-      }, 500);
+      }, 100);
```

Comment:
We started having the timeouts for bulkUpsert tests only recently, so my guess is a tiny delay is enough. If this comes up as a flapper I will fix it immediately. Is this ok?